app/isomorphic/components/breaking-news-view.js

```diff
@@ -49,7 +49,7 @@ export const BreakingNewsViewBase = ({ breakingNews = [], breakingNewsConfig = {
   }
 
   return (
-    <div styleName="base">
+    <div styleName="base" className="container">
```

Comment:
can we keep classname specific to this component, the container makes a generic name